src/main/java/ldbc/snb/datagen/DatagenParams.java

```diff
@@ -477,4 +399,84 @@ private static long findNumPersonsFromGraphalyticsScale(DegreeDistribution distr
         }
         return currentNumPersons;
     }
+
+    //TODO: add remaining degree distributions
+    public static DegreeDistribution getDegreeDistribution() {
```

Comment:
wouldn't it make more sense to do a lazy singleton initialization, than creating new instances everytime this function is called?

Reply:
hmm on second thoughts, it's simpler this way, and we'll have another change of fleshing this out once the Spark part is soldified. I would leave it as it is for now